recipe/session/sessmodels/models.go

```diff
@@ -174,6 +176,13 @@ type SessionContainer struct {
 	UpdateAccessTokenPayloadWithContext func(newAccessTokenPayload map[string]interface{}, userContext supertokens.UserContext) error
 	GetTimeCreatedWithContext           func(userContext supertokens.UserContext) (uint64, error)
 	GetExpiryWithContext                func(userContext supertokens.UserContext) (uint64, error)
+	MergeIntoAccessTokenPayload         func(accessTokenPayloadUpdate map[string]interface{}, userContext supertokens.UserContext) error
+
+	AssertClaims     func(claimValidators []*claims.SessionClaimValidator, userContext supertokens.UserContext) error
+	FetchAndSetClaim func(claim *claims.TypeSessionClaim, userContext supertokens.UserContext) error
+	SetClaimValue    func(claim *claims.TypeSessionClaim, value interface{}, userContext supertokens.UserContext) error
+	GetClaimValue    func(claim *claims.TypeSessionClaim, userContext supertokens.UserContext) (interface{}, error)
+	RemoveClaim      func(claim *claims.TypeSessionClaim, userContext supertokens.UserContext) error
 }
```

Comment:
missing changes to SignUpPOST API interface function

Reply:
did u mean signOutPOST? that's done.